WCSim.mac

```diff
@@ -27,7 +27,17 @@
 #Use mPMTs settings (uncomment/delete the above)
 #/WCSim/WCgeom nuPRISM_mPMT
 #/WCSim/WCgeom nuPRISMBeamTest_mPMT
-/WCSim/WCgeom nuPRISMShort_mPMT
+#/WCSim/WCgeom nuPRISMShort_mPMT
```

Comment:
Lauren has created a new WCTE dedicated macro ([macros/nuPRISMBeamTest.mac](https://github.com/WCTE/WCSim/blob/nuPRISM/develop/macros/nuPRISMBeamTest.mac)) to simplify and clarify for new users. could you move this to there (and keep just general commands that would work for any geometry here)?